test/features/email/presentation/extensions/calendar_event_extension_test.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/calendar_event.dart';
+import 'package:tmail_ui_user/features/email/presentation/extensions/calendar_event_extension.dart';
+import 'package:tmail_ui_user/main/utils/app_utils.dart';
+
+void main() {
+  group('calendar event extension text', () {
+    final dateTime = DateTime(2022, 12, 31, 13, 39, 00, 00, 00);
+    final sundayInCurrentDateLocale = AppUtils.getCurrentDateLocale().daysLong[5];
+    final decemberInCurrentDateLocale = AppUtils.getCurrentDateLocale().monthsLong.last;
+    final meridiemIndicator = AppUtils.getCurrentDateLocale().pm;
+    final expectedFormattedDateTime = '$sundayInCurrentDateLocale, $decemberInCurrentDateLocale 31, 2022 01:39 $meridiemIndicator';
+    final expectedFormattedTime = '01:39 $meridiemIndicator';
```

Comment:
Expected value should leave a specific value

Reply:
done